src/kernel/vmm.zig

```diff
@@ -581,41 +584,41 @@ fn runtimeTests(comptime Payload: type, vmm: VirtualMemoryManager(Payload), mem_
     }
 
     // Make sure all blocks before the kernel code are not set
     while (vaddr < tty_addr) : (vaddr += BLOCK_SIZE) {
-        const set = vmm.isSet(vaddr) catch |e| panic(@errorReturnTrace(), "Failed to check if tty address {x} is set: {x}", .{ vaddr, e });
-        if (set) panic(null, "Address before tty was set: {x}", .{vaddr});
+        const set = vmm.isSet(vaddr) catch |e| panic(@errorReturnTrace(), "FAILURE: Failed to check if tty address 0x{X} is set: 0x{X}\n", .{ vaddr, e });
+        if (set) panic(null, "FAILURE: Address before tty was set: 0x{X}\n", .{vaddr});
     }
     // Make sure all blocks associated with the kernel code are set
     while (vaddr < tty_addr + tty_buff_size) : (vaddr += BLOCK_SIZE) {
-        const set = vmm.isSet(vaddr) catch |e| panic(@errorReturnTrace(), "Failed to check if tty address {x} is set: {x}", .{ vaddr, e });
-        if (!set) panic(null, "Address for tty was not set: {x}", .{vaddr});
+        const set = vmm.isSet(vaddr) catch |e| panic(@errorReturnTrace(), "FAILURE: Failed to check if tty address 0x{X} is set: 0x{X}\n", .{ vaddr, e });
+        if (!set) panic(null, "FAILURE: Address for tty was not set: 0x{X}\n", .{vaddr});
     }
 
     // Make sure all blocks before the kernel code are not set
     while (vaddr < v_start) : (vaddr += BLOCK_SIZE) {
-        const set = vmm.isSet(vaddr) catch |e| panic(@errorReturnTrace(), "Failed to check if kernel code address {x} is set: {x}", .{ vaddr, e });
-        if (set) panic(null, "Address before kernel code was set: {x}", .{vaddr});
+        const set = vmm.isSet(vaddr) catch |e| panic(@errorReturnTrace(), "FAILURE: Failed to check if kernel code address 0x{X} is set: 0x{X}\n", .{ vaddr, e });
+        if (set) panic(null, "FAILURE: Address before kernel code was set: 0x{X}\n", .{vaddr});
     }
     // Make sure all blocks associated with the kernel code are set
     while (vaddr < v_end) : (vaddr += BLOCK_SIZE) {
-        const set = vmm.isSet(vaddr) catch |e| panic(@errorReturnTrace(), "Failed to check if kernel code address {x} is set: {x}", .{ vaddr, e });
-        if (!set) panic(null, "Address for kernel code was not set: {x}", .{vaddr});
+        const set = vmm.isSet(vaddr) catch |e| panic(@errorReturnTrace(), "FAILURE: Failed to check if kernel code address 0x{X} is set: 0x{X}\n", .{ vaddr, e });
+        if (!set) panic(null, "FAILURE: Address for kernel code was not set: 0x{X}\n", .{vaddr});
     }
 
     // Make sure all blocks after the kernel code are not set
     while (vaddr < vmm.end - BLOCK_SIZE) : (vaddr += BLOCK_SIZE) {
-        const set = vmm.isSet(vaddr) catch |e| panic(@errorReturnTrace(), "Failed to check if address after {x} is set: {x}", .{ vaddr, e });
-        if (set) panic(null, "Address after kernel code was set: {x}", .{vaddr});
+        const set = vmm.isSet(vaddr) catch |e| panic(@errorReturnTrace(), "Failed to check if address after 0x{X} is set: 0x{X}", .{ vaddr, e });
```

Comment:
The second set of braces in each message should actually be `{}` since they correspond to the error not a number.